pkg/repository/v1_repository.go

```diff
@@ -188,6 +200,10 @@ func (r *V1Repository) selectVersion(id string, versions map[string]v1manifest.V
 		var latest string
 		var latestItem v1manifest.VersionItem
 		for version, item := range versions {
+			if v0manifest.Version(version).IsNightly() {
```

Comment:
Can we inline the 'is nightly' check here so we don't need to use v0manifest.Version?

Reply:
I think that we should add to manifests.md that a nightly version must contain the substring `nightly`. We could also check that in `Component.IsValid`

Reply:
Good catch by the way, I did not think about this issue with nightlies.